src/types/index.ts

```diff
@@ -77,6 +77,38 @@ export interface DepositApi {
   withdraw(userAddress: string, tokenAddress: string, txOptionalParams?: TxOptionalParams): Promise<Receipt>
 }
 
+export interface EpochTokenLocker extends Contract {
+  clone(): EpochTokenLocker
+
+  methods: {
+    hasCreditedBalance(user: string, token: string, batchId: number | string | BN): TransactionObject<boolean>
+
+    BATCH_TIME(): TransactionObject<string>
```

Comment:
Why CAPs rather than camelCase?

Reply:
It's a contract method that corresponds to [a field on the contract](https://github.com/gnosis/dex-contracts/blob/671a77dd1c262563320cb538e81defbb7b1dc078/contracts/EpochTokenLocker.sol#L38). We can't change that